module/os/linux/spl/spl-zlib.c

```diff
@@ -202,7 +202,7 @@ spl_zlib_init(void)
 	zlib_workspace_cache = kmem_cache_create(
 	    "spl_zlib_workspace_cache",
 	    size, 0, NULL, NULL, NULL, NULL, NULL,
-	    KMC_KVMEM);
+	    KMC_KVMEM | KMC_RECLAIMABLE);
```

Comment:
Is the `zlib_workspace_cache` memory really reclaimable?  It looks like it's just transient memory that gets used during `z_compress_level()`.

Reply:
It is not reclaimable indeed, but it is very short-lived, and in few places it is suggested to be treat it the same. I just saved on another define:
```
#define SLAB_TEMPORARY          SLAB_RECLAIM_ACCOUNT    /* Objects are short-lived */
```

Reply:
Got it, thanks